opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -262,8 +262,7 @@ def create_time_series_list(self, v_data, option_resource_type,
             secs = point.interval.end_time.seconds
             point.interval.end_time.nanos = int((timestamp_end - secs) * 10**9)
 
-            if type(agg) is not aggregation.aggregation_data.\
-                    LastValueAggregationData:  # pragma: NO COVER
+            if aggregation_type is not aggregation.Type.LASTVALUE:
```

Comment:
Do we skip `LastValue` aggregations here because they don't cover a range, so it's okay if `start` and `end` are the same? I'd like to add a comment if we know why this is done.

Reply:
No idea from previous PRs, you may know better what stackdriver expects here.